### DISCUSSION TITLE
Cockroachdb migrate - rowid error

### DISCUSSION THREAD

**Discussion by hanisaf** [directus/directus#16105] [Archive]
I'm having an issue after copying the database from one cockroachdb database to another. After logging in, a series of errors about updating the last page of the user show up. The first error which is the root of the issue has to do with rowid being NaN. If I know where the argument $1 is coming from, I can perhaps set it manually in the database. Or perhaps there is a better way. I appreciate help guiding me how to fix it.

"error: select "directus_users"."email", "directus_users"."first_name", "directus_users"."last_name", "directus_users"."last_page", "directus_users"."rowid" from "directus_users" where "directus_users"."rowid" = $1 order by "directus_users"."rowid" asc limit $2 - error in argument for $1: strconv.ParseInt: parsing "NaN": invalid syntax at Parser.parseErrorMessage

**Comment by connorwinston** [comment]
@hanisaf Is this an error you are having in Directus or an error you are getting when dumping/restoring data between CockroachDB Databases?

**Reply by hanisaf** [reply]
The issue is resolved now. It is not a Directus error but rather related to duplicating the database. 

Initially, I used a tool (DBeaver) to copy the database but later I performed database replication using Cockroachdb built-in backup/restore commands and it worked fine.

**Reply by connorwinston** [reply]
Ok great!
